unstructured/metrics/doctype_aggregation.py

```diff
@@ -66,7 +66,7 @@ def measure_holistic_eval_cct(
     weights: Tuple[int, int, int],
 ) -> None:
     export_dir = "result_doctype_aggregate"
-    measure_text_edit_distance(
+    measure_text_extraction_accuracy(
```

Comment:
when will this function be passed `visualize` as input (here or in the test, for example)? I see that the default value is false, so is it getting used yet?

Reply:
Referring to this click option
```
@click.option(
    "--visualize",
    is_flag=True,
    show_default=True,
    default=False,
    help="Add the flag to show progress bar.",
)
```
The `--visualize` param will be passed in as a flag. Meaning you can use by adding `--visualize` to the command line that calls the script. It is not being used yet. The main purpose of this additional param/flag is to track the progress of the `evaluate.py` when the list of files to evaluate against are large so we don't get stuck staring at blank terminal for like an hour. I find that quite frustrating when I experienced. So to answer your question, no it is not getting used yet anywhere in this repo or script since the evaluating list is relatively small that it won't get stuck at processing for long.